### PR TITLE
Cart journey

### PR DESCRIPTION
**Pull request by areebali99**
### Goal:
Add to cart Journey

### Task:
merge conflicts resolved of card and drawer add to cart journey

### CODE REVIEW

File: src/Features/Customer/Dashboard/index.js [hunk 1]
```diff
@@ -1,42 +1,31 @@
...
+
+  function toggleDrawer() {
+    setOpenDrawer(!isDrawerOpen);
+  }
+
+  const openDrawer = () => {
+    setOpenDrawer(true);
+  };
+
+  const closeDrawer = () => {
```
Comment (ShahirZain) [inline src/Features/Customer/Dashboard/index.js:26]: why do we need to write explicit function for open and close drawer, while we have toggleDrawer???

File: src/Features/Customer/Dashboard/index.js [hunk 2]
```diff
@@ -46,32 +35,32 @@ function Dashboard() {
         </Grid>
         <Grid item md={9}>
           <Box>
-            <div className={header}>
-              <div className={orderDetails}>
+            <Header>
+              <OrderDetails>
                 <ArrowBackIcon fontSize="large" style={{ color: '#e91e63' }} />
```
Comment (ShahirZain) [inline src/Features/Customer/Dashboard/index.js:40]: remove inline styling . . .and  Color with color string

File: src/Features/Customer/Dashboard/index.js [hunk 3]
```diff
@@ -46,32 +35,32 @@ function Dashboard() {
...
                   #023025
-                </Typography>
-              </div>
+                </OrderRef>
+              </OrderDetails>
 
-              <div className={userInfo}>
+              <UserInfo>
                 <Typography variant="h4">
                   <b> Arham Ahmed</b>
```
Comment (ShahirZain) [inline src/Features/Customer/Dashboard/index.js:53]: why did you use hard coded data?

File: src/components/CardMenus/CardMenus.js [hunk 4]
```diff
@@ -8,32 +8,35 @@ import CommonCard from './CommonCard';
 const useStyles = makeStyles((theme) => ({
```
Comment (ShahirZain) [inline src/components/CardMenus/CardMenus.js:8]: remove make style . .  replace with it Styled Component

File: src/components/CardMenus/CardMenus.js [hunk 5]
```diff
@@ -8,32 +8,35 @@ import CommonCard from './CommonCard';
...
 }));
 
-const CardMenu = () => {
+const CardMenu = ({ foodType }) => {
   const classes = useStyles();
+  // const cart = useSelector((state) => state.addtocartReducers.cart);
 
   const { control } = classes;
-
+  console.log('food', foodType);
```
Comment (ShahirZain) [inline src/components/CardMenus/CardMenus.js:20]: remove console

File: src/components/CardMenus/CardMenus.js [hunk 6]
```diff
@@ -8,32 +8,35 @@ import CommonCard from './CommonCard';
...
-              key={id}
-              buttonText="Add to Cart"
-              img={img}
-              name={name}
-              price={price}
-              resturantName={resturantName}
-            />
-          );
+      <Grid className={classes.control} container elevation={3} spacing={3}>
+        {UserData.map((usedata, index) => {
```
Comment (ShahirZain) [inline src/components/CardMenus/CardMenus.js:24]: userData is not relevant name

File: src/components/CardMenus/CardMenus.js [hunk 7]
```diff
@@ -8,32 +8,35 @@ import CommonCard from './CommonCard';
...
+          const { id, name, price, type, resturantName, img } = usedata;
```
Comment (ShahirZain) [inline src/components/CardMenus/CardMenus.js:25]: we can destructure on parameter level

File: src/components/CardMenus/CardMenus.js [hunk 8]
```diff
@@ -8,32 +8,35 @@ import CommonCard from './CommonCard';
...
+          console.log('type', type);
```
Comment (ShahirZain) [inline src/components/CardMenus/CardMenus.js:26]: there should be no console in project

File: src/components/CardMenus/CardMenus.js [hunk 9]
```diff
@@ -8,32 +8,35 @@ import CommonCard from './CommonCard';
...
+          if (type === foodType) {
```
Comment (ShahirZain) [inline src/components/CardMenus/CardMenus.js:27]: i did not get this if check please make it clear. What is going on here

File: src/components/CardMenus/CommonCard.js [hunk 10]
```diff
@@ -1,125 +1,39 @@
...
-          }
-        />
-        <CardContent className={content}>
+      <CardRoot>
+        <ImageDiv src={img} title={name} />
+
+        <HeaderCard subheader={<h4>{resturantName}</h4>} title={<FoodTitle variant="h2">{name}</FoodTitle>} />
+
+        <Content>
           <div style={{ marginTop: '20px', display: 'flex' }}>
```
Comment (ShahirZain) [inline src/components/CardMenus/CommonCard.js:21]: remove inline styling

**Review by ShahirZain — CHANGES_REQUESTED** [review]

**Review by SiddiquiMonis401 — APPROVED** [review]

**Review by ShahirZain — APPROVED** [review]

**Pull request merged**